### DISCUSSION TITLE
Solax adapter box G2

### DISCUSSION THREAD

**Discussion by jakubhepnar** [wills106/homeassistant-solax-modbus#930] [Ideas]
Hello,

thank you for the great integration into home assistant.

I would like to ask if it is possible to add integration for the Solax Adapter box G2 for controlling 1-4 dry contacts?

https://www.solaxpower.com/solax-adapter-box/

**Comment by Technojunk94** [comment]
i am looking for the same!
Is this possible @infradom ?

**Comment by Xellzul** [comment]
I am also interested in this, is there any official documentation? Is it possible to control this from Inverter? It doesn't look like it has Modbus TCP

**Comment by Technojunk94** [comment]
Only possible to controle with app yet.
I hope this becomes a function to controller by HA

Met vriendelijke groet,
Niels Kloppenburg

Op 7 nov 2024 om 13:25 heeft Xellzul ***@***.***> het volgende geschreven:

﻿

I am also interested in this, is there any official documentation? Is it possible to control this from Inverter? It doesn't look like it has Modbus TCP
